### DISCUSSION TITLE
Do something after all tests have passed

### DISCUSSION THREAD

**Discussion by realadilet** [laravel/framework#38101] [Q&A]
Have any ideas how to do this?

**Comment by oliworx** [comment]
why not just create a script around the test command, something like
```
#!/bin/bash
vendor/bin/phpunit
cleanup.sh
do-whatever.sh
...
```

or make an artisan command that calls phpunit.
I think there are a lot of ways to do this

**Reply by realadilet** [reply]
I need call method in one session with phpunit

**Reply by MahbodHastam** [reply]
Hmm, you can move that function to the end of the class :P
